core/textile/client.go

```diff
@@ -172,6 +178,52 @@ func (tc *textileClient) SetUc(uc UsersClient) {
 	tc.uc = uc
 }
 
+func (tc *textileClient) SetMb(mb Mailbox) {
```

Comment:
I see this is only used for tests, but feels wrong adding more stuff to the textile client public interface. Maybe you can change the package name of `textile_test.go` to `textile` to get access to private fields and simply do `tc.mb = mockMb`. Or maybe it's not too bad as long as `textile.go` doesn't expose that method.

Reply:
Fixed in [4f496ce](https://github.com/FleekHQ/space-daemon/pull/134/commits/4f496cebd9add9c7d6cd86c2194408a4995bf3a3). I removed them both by making them init parameters and injecting them from the top.  The helpers to create the clients still in textile package but exposed one of them so we can call it from `app.go` and inject.  Also the weird part is I made `mb` optional in the init because it gets set later, but this way I can inject during test as well. Not perfect but better than before I think.